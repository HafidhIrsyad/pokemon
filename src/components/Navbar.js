import { Link } from 'react-router-dom';

function Navbar () {
  return (
    <header className="h-full bg-green-500 ">
      <nav className="flex justify-around h-24 items-center text-white">
        <div>
          <Link to="/" className="text-3xl">Pokemon</Link>
        </div>
        <ul className="flex justify-between">
          <li>
            <Link to="/favorite" className="text-lg">Favorite</Link>
          </li>
          <li>
            <a 
              href="https://github.com/mazipan/graphql-pokeapi" 
              target="_blank"
              rel="noreferrer" 
              className="text-lg ml-10">Pokemon Resource by Irfan Maulana
            </a>
          </li>
        </ul>
      </nav>
    </header>
  );
}

export default Navbar;