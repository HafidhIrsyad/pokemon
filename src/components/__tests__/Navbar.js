import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Navbar from '../Navbar';

describe('src/components/Navbar', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Navbar />);
    expect(tree).toMatchSnapshot();
  });
});
