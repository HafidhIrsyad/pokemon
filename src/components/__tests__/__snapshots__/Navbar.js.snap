// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Navbar render 1`] = `
<header
  class="border-b md:flex md:items-center md:justify-around p-4 pb-0 shadow-lg md:pb-4 bg-green-500 text-white"
>
  <div
    class="flex items-center justify-between mb-4 md:mb-0"
  >
    <h1
      class="leading-none text-2xl text-grey-darkest"
    >
      <Link
        class="no-underline text-grey-darkest hover:text-black"
        to="/"
      >
        Pokemon
      </Link>
    </h1>
  </div>
  <nav>
    <ul
      class="list-reset md:flex md:items-center"
    >
      <li
        class="md:ml-4"
      >
        <Link
          class="block no-underline hover:underline py-2 text-grey-darkest hover:text-black md:border-none md:p-0"
          to="/favorite"
        >
          Favorite
        </Link>
      </li>
      <li
        class="md:ml-4"
      >
        <a
          class="border-t block no-underline hover:underline py-2 text-grey-darkest hover:text-black md:border-none md:p-0"
          href="https://github.com/mazipan/graphql-pokeapi"
          rel="noreferrer"
          target="_blank"
        >
          Pokemon Resource by Irfan
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
